src/tests/ftest/util/dmg_utils.py

```diff
@@ -44,72 +44,266 @@ def __init__(self, path):
+
+            def __init__(self):
+                """Create a dmg pool get-acl command object."""
+                super(
+                    DmgCommand.PoolSubCommand.GetAclSubCommand,
+                    self).__init__(
+                        "/run/dmg/pool/get-acl/*", "get-acl")
+                self.pool = FormattedParameter("--pool={}", None)
+
+        class OverwriteAclSubCommand(CommandWithParameters):
+            """Defines an object for the dmg pool overwrite-acl command."""
+
+            def __init__(self):
+                """Create a dmg pool overwrite-acl command object."""
+                super(
+                    DmgCommand.PoolSubCommand.OverwriteAclSubCommand,
+                    self).__init__(
+                        "/run/dmg/pool/overwrite-acl/*", "overwrite-acl")
+                self.pool = FormattedParameter("--pool={}", None)
+                self.acl_file = FormattedParameter("-a {}", None)
+
+        class UpdateAclSubCommand(CommandWithParameters):
+            """Defines an object for the dmg pool update-acl command."""
+
+            def __init__(self):
+                """Create a dmg pool update-acl command object."""
+                super(
+                    DmgCommand.PoolSubCommand.UpdateAclSubCommand,
+                    self).__init__(
+                        "/run/dmg/pool/update-acl/*", "update-acl")
+                self.pool = FormattedParameter("--pool={}", None)
+                self.acl_file = FormattedParameter("-a {}", None)
+                self.entry = FormattedParameter("-e {}", None)
+
+    class StorageSubCommand(CommandWithSubCommand):
+        """Defines an object for the dmg storage sub command."""
 
         def __init__(self):
-            """Create a dmg Command object."""
-            super(DmgCommand.DmgCreateSubCommand, self).__init__(
-                "/run/dmg/create/*", "create")
-            self.group = FormattedParameter("-g {}")
-            self.user = FormattedParameter("-u {}")
-            self.acl_file = FormattedParameter("-a {}")
-            self.scm_size = FormattedParameter("-s {}")
-            self.nvme_size = FormattedParameter("-n {}")
-            self.ranks = FormattedParameter("-r {}")
-            self.nsvc = FormattedParameter("-v {}")
-            self.sys = FormattedParameter("-S {}")
-
-    class DmgDestroySubCommand(CommandWithParameters):
-        """Defines an object representing a destroy sub dmg command."""
+            """Create a dmg storage subcommand object."""
+            super(DmgCommand.StorageSubCommand, self).__init__(
+                "/run/dmg/storage/*", "storage")
+
+        def get_sub_command_class(self):
+            """Get the dmg storage sub command object."""
+            if self.sub_command.value == "format":
+                self.sub_command_class = self.FormatSubCommand()
+            elif self.sub_command.value == "prepare":
+                self.sub_command_class = self.PrepareSubCommand()
```

Comment:
(pylint-redefined-variable-type) Redefinition of self.sub_command_class type from util.dmg_utils.DmgCommand.StorageSubCommand.FormatSubCommand to util.dmg_utils.DmgCommand.StorageSubCommand.PrepareSubCommand